### ISSUE TITLE
Offset from parent Safari

### ISSUE THREAD

**Issue by surielmx** [superman66/vue-highcharts#20]
Hi, when I go to this page, the chart looks fine, but when reload the page, the chart width is offset from parent too much. Only happen in Safari

![screen shot 2017-10-10 at 7 49 32 pm](https://user-images.githubusercontent.com/1803166/31417348-804ef898-adf4-11e7-9e58-51c1ef451bf1.png)

![screen shot 2017-10-10 at 7 52 23 pm](https://user-images.githubusercontent.com/1803166/31417361-91b885cc-adf4-11e7-8709-72d3794f0b2c.png)

Greetings!

**Comment by superman66** [comment]
have you set the chart width value?

**Comment by surielmx** [comment]
Yes, but I don't need the width with px, it supposed to fit to width of parent container. Previous version of this project was with angular2 and don't have any issue with highcharts angular's version

Greetings!
